### ISSUE TITLE
[Feature suggestion] Custom datapacks for all recipes

### ISSUE THREAD

**Issue by Rimevel** [elucent/eidolon#19]
I would love to see more datapack opportunities for this mod to easier integrate with other when making modpacks.

Adding new or removing existing...

Crucible recipes
Altar objects and their power/capacity levels
Magic workbench recipes
Ritual Recipes

**Comment by Rimevel** [comment]
Also maybe allow for custom recipes to show up in the book? For maximum immersion :)

**Comment by NillerMedDild** [comment]
Elucent do you have any stance on this? I'd like to know whether you plan to implement this or not, as it's critical in integrating Eidolon in an Expert modpack :)

**Comment by elucent** [comment]
It will be in the next major release, but I don't have an ETA on that as there are several major systems I'd like to introduce in it as well. So, definitely planned, I actually have the implementation working for magic workbench recipes already, but it is probably a while out from being public. If you have a pressing need as a modpack developer I would be willing to talk about it, it might be possible to push it out early.

**Comment by NillerMedDild** [comment]
Awesome!

> If you have a pressing need as a modpack developer I would be willing to talk about it, it might be possible to push it out early.

I would like to use Eidolon in Enigmatica 6: Expert, and being able to tweak and add recipes is pretty crucial in that, but I don't know if that a qualifies as a pressing need. I'm just glad this is a planned feature, I think I'm able to make due until you have time to implement it :D

**Comment by elucent** [comment]
Yeah, I mean, let me know if you need the feature right away. Otherwise it'll probably be out in general in...a month or two maybe? No promises, but I'd hope 0.3 won't take longer than that.

**Comment by NillerMedDild** [comment]
Any updates on this? :) Datapack support would be very nice.

**Comment by elucent** [comment]
@NillerMedDild For various reasons I haven't been able to make a lot of progress on the next release...but someone has separated out my WIP implementation (originally distributed through a beta in my discord) and turned it into an independent mod: https://www.curseforge.com/minecraft/mc-mods/eidolon-recipes

I recommend you use this until the next patch.

**Comment by NillerMedDild** [comment]
Alright I will, thanks 👍

**Comment by NillerMedDild** [comment]
I'm afraid Eidolon Recipes doesn't provide tag support in recipes, cannot remove recipes, and is config-only, so it's not really a viable alternative tbh. Thanks anyway :)
